apps/system/js/screen_manager.js

```diff
@@ -206,6 +206,14 @@
 
         case 'sleep':
           this.turnScreenOff(true, 'powerkey');
+          var lock_enabled = window.lockScreen.enabled;
```

Comment:
Please note the convention in this file is camel cases. See `this._deviceLightEnabled` above.